app/views/users/_user.html.erb

```diff
@@ -0,0 +1,9 @@
+<section>
+    <div class="img">
+        <%= image_tag user.photo %>
+    </div>
+    <div class="user-post">
+     <h2><%= link_to user.name, user %></h2>
+         <p>Number of posts <%= user.posts_counter ? user.posts_counter : 0 %></p>
+    </div>
+</section>
```

Comment:
- [ ]  It will be good if posts will be clickable and show the post. Kindly add it.
For see the post ı have to write the this URL `users/1/posts/1`